### ISSUE TITLE
CURL error: error setting certificate verify locations:   CAfile: /etc/pki/tls/certs/ca-bundle.crt   CApath: none

### ISSUE THREAD

**Issue by chlsl** [cmla/tsd#22]
I just upgraded tsd and now I get the following error when downloading an image:
```
$  python3 tsd/get_sentinel2.py --lat 49.037670 --lon 3.942441 --start-date 2017-09-19 --end-date 2017-09-21 --band B02 B03 B04 B08 --api scihub --product-type L1C -o ../data/champagne-test/S2/L1C
Found 1 images
Building 1 gcloud download urls... 1 / 1
Downloading 4 crops (1 images with 4 bands)... Traceback (most recent call last):
  File "rasterio/_base.pyx", line 216, in rasterio._base.DatasetBase.__init__
  File "rasterio/_shim.pyx", line 67, in rasterio._shim.open_dataset
  File "rasterio/_err.pyx", line 205, in rasterio._err.exc_wrap_pointer
rasterio._err.CPLE_HttpResponseError: CURL error: error setting certificate verify locations:   CAfile: /etc/pki/tls/certs/ca-bundle.crt   CApath: none

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "tsd/get_sentinel2.py", line 366, in <module>
    parallel_downloads=args.parallel_downloads)
  File "tsd/get_sentinel2.py", line 292, in get_time_series
    download(images, bands, aoi, mirror, out_dir, parallel_downloads)
  File "tsd/get_sentinel2.py", line 171, in download
    nb_workers=parallel_downloads)
  File "/home/hessel/kayrros/tsd/tsd/parallel.py", line 75, in run_calls
    outputs.append(r.get(timeout))
  File "/usr/lib/python3.6/multiprocessing/pool.py", line 644, in get
    raise self._value
  File "/usr/lib/python3.6/multiprocessing/pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "/home/hessel/kayrros/tsd/tsd/utils.py", line 192, in crop_with_rasterio
    with rasterio.open(inpath) as src:
  File "/home/hessel/.local/lib/python3.6/site-packages/rasterio/env.py", line 445, in wrapper
    return f(*args, **kwds)
  File "/home/hessel/.local/lib/python3.6/site-packages/rasterio/__init__.py", line 219, in open
    s = DatasetReader(path, driver=driver, sharing=sharing, **kwargs)
  File "rasterio/_base.pyx", line 218, in rasterio._base.DatasetBase.__init__
rasterio.errors.RasterioIOError: CURL error: error setting certificate verify locations:   CAfile: /etc/pki/tls/certs/ca-bundle.crt   CApath: none
```

I investigated a bit and found that this was introduced at commit 5c3c7ad87e2340d35e97092ece8055c3941c2cdc.

Fortunately this can be fixed with `export CURL_CA_BUNDLE=/etc/ssl/certs/ca-certificates.crt` (https://stackoverflow.com/q/3160909).

Maybe we could change this path in tsd?

**Comment by glostis** [comment]
Hi @chlsl,

It's my fault if you get the error, because I authored the offending commit 😃 

In this commit, I have replaced all uses of `GDAL` by the use of the more pythonic `rasterio`.

The error you encounter is due to the platform on which the `rasterio` `manylinux` wheels are built.
Unfortunately, it seems that the only viable option is to warn the Ubuntu users about this on `tsd`'s README.

See here for the detailed explanation on `rasterio`'s README: https://github.com/mapbox/rasterio#ssl-certs

And see here for a comment where `rasterio`'s maintainer explains why he can't/won't fix it:
https://github.com/mapbox/rasterio/issues/1737#issuecomment-524001890
